### DISCUSSION TITLE
More name aliases

### DISCUSSION THREAD

**Discussion by tknx** [jellyfin/jellyfin#8545] [General]
If you watch foreign media, it would be nice to have more options for name aliases for shows and media.

So let's take an example like an anime show.

1. Original Title in Japanese: やはり俺の青春ラブコメはまちがっている
2. Original Title in Romaji: Yahari Ore no Seishun Love Comedy wa Machigatteiru.
3. Shortened abbreviation title: Oregairu
4. English Title: My Teen Romantic Comedy SNAFU

Ideally you could add as many titles as you want and just select one as the "main" one to be displayed.

I fully recognize this is probably not the highest value item out there though!

**Comment by Arutosio** [comment]
can I set the names in Japanese and everything else in English?
